lib/features/rules_filter_creator/presentation/rules_filter_creator_controller.dart

```diff
@@ -160,7 +169,13 @@ class RulesFilterCreatorController extends BaseMailboxController {
           controller: TextEditingController(),
         );
         listRuleConditionValueArguments.add(newRuleConditionValueArguments);
-        emailRuleFilterActionSelected.value = EmailRuleFilterAction.moveMessage;
+        isShowAddAction.value = true;
+        RuleFilterActionArguments newRuleFilterAction = RuleFilterActionArguments(
+          action: null,
+          detailed: null,
+          error: '',
+        );
```

Comment:
Should be create empty constructor  in  RuleFilterActionArguments by use factory
```suggestion

factory RuleFilterActionArguments.empty() => RuleFilterActionArguments(action: null, detailed: null, error: '' );

```